js/private/js_run_binary.bzl

```diff
@@ -107,6 +110,21 @@ def js_run_binary(
 
             This makes node binaries match the expected bazel paradigm.
 
+        run_from_execroot: Run the `tool` js_binary entry_point script that is in the execroot of the output tree
+            instead of the copy that is in runfiles.
+
+            Runfiles of `tool` are all hoisted to `srcs` of the underlying `run_binary` so they are included as execroot
+            inputs to the action.
+
+            Running the entry point out of the execroot output tree means that there is no conflicting runfiles
```

Comment:
"means that there is no conflicting runfiles" => "means there are no conflicting runfiles"?

Reply:
🍔